java/client/src/main/java/glide/api/commands/GeospatialIndicesBaseCommands.java

```diff
@@ -57,4 +57,23 @@ CompletableFuture<Long> geoadd(
      * }</pre>
      */
     CompletableFuture<Long> geoadd(String key, Map<String, GeospatialData> membersToGeospatialData);
+
+    /**
+     * Return the positions (longitude,latitude) of all the specified members of the geospatial index
+     * represented by the sorted set at <code>key
+     * </code>.<br>
+     *
+     * @see <a href="https://valkey.io/commands/geopos">valkey.io</a> for more details.
+     * @param key The key of the sorted set.
+     * @param members The members for which to get the positions.
+     * @return A list of positions (longitude and latitude) corresponding to the given members. If a
+     *     member does not exist, its position will be Null.
```

Comment:
I think we typically don't capitalize Null, but put it in code tags:
```suggestion
     *     member does not exist, its position will be <code>null</code>.
```